include/ck_tile/ops/gemm/kernel/universal_gemm_kernel.hpp

```diff
@@ -325,10 +325,16 @@ struct UniversalGemmKernel
     {
         __device__ SplitKBatchOffset(const KernelArgs& kargs, const std::size_t k_id = blockIdx.z)
         {
-            constexpr auto K1   = GemmPipeline::BlockGemmShape::WarpTile::at(number<2>{});
+            constexpr auto K1   = TilePartitioner::KPerBlock;
```

Comment:
@aosewski can you take a look? Im not sure if we want such solution. In the old ck we processed it with padding